### DISCUSSION TITLE
Second click on tab to hide.

### DISCUSSION THREAD

**Discussion by zaka221** [bladewindui/ui#108] [Q&A]
Like in title, there is a way to do it or it already exists and I doing somethink wrong.
Thank you, I learn  a lot from this project.

**Comment by mkocansey** [comment]
@zaka221 which component are you referring to? If you are referring to the [tab component](https://bladewindui.com/component/tag) then no, this does not exist. That is in fact, not how tabs work. You don't typically click on tab headings to hide them. 

Take for example the Github box where you type comments. There are 2 tabs. **Write** and **Preview**.
![Screenshot 2023-03-23 at 12 31 13](https://user-images.githubusercontent.com/1946332/227204388-e2394d2f-7373-42c8-91d0-f34f54fa3f2f.png)

Clicking on **Write** two times will not hide it.
